docs/user/OIDC/OAuth2OIDC-keycloak.md

```diff
@@ -1,9 +1,310 @@
 # Configuring Keycloak as an OIDC provider
+(created May 17th 2021)
 
-This document explains how to configure Keycloak as an OIDC provider (check general information and pre-requisites for [using an OAuth2/OIDC Provider with Kubeapps](../using-an-OIDC-provider.md)).
+This document explains how to configure Keycloak as an IDP + OIDC provider (check general information and pre-requisites for [using an OAuth2/OIDC Provider with Kubeapps](../using-an-OIDC-provider.md)).
+It covers the installation and documentation for Kubeapps interacting with two AWS clusters. 
 
-In the case of Keycloak, you can find the parameters in the Keycloak admin console:
+The base installation is done using the bitnami chart for Keycloak.
```

Comment:
```suggestion
The base installation is done using the [bitnami chart for Keycloak](https://github.com/bitnami/charts/tree/master/bitnami/keycloak).
```

Reply:
i will make a change locally to include the chart versions